test/pg-cdc/pg-cdc.td

```diff
@@ -148,7 +148,8 @@ CREATE PUBLICATION another_publication FOR TABLE another_schema.another_table;
 
 #
 # Test that slots created for replication sources are deleted on DROP
-$ postgres-verify-slot connection=postgres://postgres:postgres@postgres slot=materialize_% active=false
+# TODO: enable once we land https://docs.google.com/document/d/19i1lBaTKY4-G9Vi2_2-FpRzA1RecYuOCcOk0KMnDacI/edit#heading=h.a3wek7c0fc48
```

Comment:
Can we please link an issue? It will allow us to be notified when the issue is closed and the TODO is still present.

Reply:
done